README.md

```diff
@@ -95,6 +96,28 @@ Subsystem by running the following in PowerShell as an administrator
 Restart-Service LxssManager*
 ```
 
+### Local Development Login
+
+To log in to the system once locally deployed, simply enter a username in the designated field and click the "Login" button.
+
+#### Users & user types
+
+The following table lists the available usernames and their corresponding user types, as well as their associated permissions:
+
+| Username | Type of User | User Permission Scope |
+|:--------:|:-------------:|:----------------:|
+|developer | Developer user| developer |
+|researcher | Researcher user| researcher |
+|user04 | Pre-clinical-intake user| pre-clinical-intake |
+|user07 | Bioinformatics-section user| bioinformatics-section-user |
+
+Please note that the permissions listed in the table are subject to change.
+
+#### Login screenshots
+
+![Login page](./docs/figures/login.png "Login page")
+![User logged in](./docs/figures/logged-in.png "Developer user logged in")
+
```

Comment:
This can either be included in this task or a new one.  Another subsection under this one is perfect for describing how the login screens differ between local and production deployments.  The production deployments of Rosalution strip the streamlined development login and only supports the configured CAS login.  This is a rather important feature & limitation.

Reply:
I think this would be a good as its own task as right now I am locked out of the VPN to access the UAB network to get the screenshots for this, but I could add the text now for this request.

Reply:
Sounds good. I will draft another Wrike ticket during standup today.